tasmota/tasmota_xsns_sensor/xsns_05_ds18x20.ino

```diff
@@ -27,6 +27,12 @@
 
 //#define USE_DS18x20_RECONFIGURE    // When sensor is lost keep retrying or re-configure
 //#define DS18x20_USE_ID_AS_NAME      // Use last 3 bytes for naming of sensors
+      
+#define DS18x20_USE_ID_ALIAS
```

Comment:
I would suggest to move this outside so that
1. It is not defined by default (no impact of existing builds)
2. It can be defined in user_config_override.h

You may also add a commented (not enabled) line in user_config.h as a reference

Reply:
NNot now, because I'm not ready for the esp 32 board. If this function is in demand, then after being added to xsns_05_esp32_ds18x20.ino, we can take it outside.

Reply:
Add support for esp32 and move DS18x20_USE_ID_ALIAS outsde